manifest/v1alpha/alertpolicy/measurement.go

```diff
@@ -63,6 +63,16 @@ func GetExpectedOperatorForMeasurement(measurement Measurement) (v1alpha.Operato
 	}
 }
 
+// getExpectedOperatorForMeasurement returns the operator that should be paired with a given measurement.
+func getExpectedOperatorForMeasurement(measurement Measurement) (v1alpha.Operator, error) {
```

Comment:
Actually, we can leave it public to avoid breaking change if anyone except us use it.